src/framework/COCacheBox.h

```diff
@@ -0,0 +1,16 @@
+//  Created by Mathieu Dutour on 14/02/2019.
+//  
+
+#import <Foundation/Foundation.h>
+#import <JavaScriptCore/JavaScriptCore.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface COCacheBox : NSObject
```

Comment:
I have no idea what this is or what it's for. Please can it have some overview docs.